backend/Origam.Server/Controller/UIServiceController.cs

```diff
@@ -673,6 +705,27 @@ public IActionResult GetMenuId([FromBody]GetMenuInput input)
         }
         #endregion
         
+        
+        private EntityData GetWorkQueueEntityData(WorkQueueSessionStore workQueueSessionStore)
```

Comment:
Shouldn't these methods be part of `SessionStore` (and their respective descendants) class?